cmd/main.go

```diff
@@ -194,16 +195,20 @@ func main() {
 	log.Println("Starting bm service")
 
 	var openshiftVersionsMap models.OpenshiftVersions
-
 	if Options.OpenshiftVersions == "" {
 		log.Fatal("OpenShift versions is empty")
 	}
-
 	failOnError(json.Unmarshal([]byte(Options.OpenshiftVersions), &openshiftVersionsMap),
 		"Failed to parse supported openshift versions JSON %s", Options.OpenshiftVersions)
 
 	log.Println(fmt.Sprintf("Started service with OCP versions %v", Options.OpenshiftVersions))
 
+	var mustGatherImageMap = make(versions.MustGatherVersions)
```

Comment:
Just for naming consistency, maybe name the variable as mustGatherVersionsMap (instead of mustGatherImageMap).

Reply:
Fixed